documentation/common/community/blogs.md

```diff
@@ -0,0 +1,129 @@
+---
+url: blogs
+weight: 200
+---
```

Comment:
Does this file need to be in the rat excludes too because the license isn't at the start?